pop-api/src/v0/assets/pallets/assets.rs

```diff
@@ -0,0 +1,492 @@
+#![allow(dead_code)]
+
+use crate::{Balance, RuntimeCall, *};
+use ink::prelude::vec::Vec;
+use primitives::{AssetId, MultiAddress};
+use scale::{Compact, Encode};
+
+type Result<T> = core::result::Result<T, PopApiError>;
+
+/// [Pallet Assets](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/assets/src/lib.rs):
+/// 1. Dispatchables
+/// 2. Read state functions
+///
+/// 1. Dispatchables within pallet assets (TrustBackedAssets instance) that can be used via the pop api on Pop Network:
```

Comment:
```suggestion
/// 1. Dispatchables within pallet assets (TrustBackedAssets instance)
```

I dont know why we need to keep repeating pop-api and pop network everywhere when its obvious based on the crate name and the repo. If we ever need to rename things, these names are littered through the code base. Naming should be defined at the highest level only and anything beneath it just becomes implied.